Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ruby:2.5.1
 
-RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
+RUN apt-get update -qq &&\
+    apt-get install -y build-essential libpq-dev nodejs poppler-utils ffmpeg
```

Comment:
Why qpdf and ghostscript aren't needed here? And why isn't ffmpeg on Travis too?

Reply:
Sorry I made a mistake adding `qpdf` and `ghostscript`, I'll remove them from travis.

[Here] it's specified to use `ffmpeg` and `poppler` for video and PDF previews, though I'm not sure that those are specifically a must, I think it's good to opt for the ones the gem says.

In Travis, the tools are needed because in #66 there are some tests to cover the previews. I'll update the Travis file and cherry-pick the commits again to check if they work as expected